README.md

```diff
@@ -1,47 +1,84 @@
-# Geek Sessions website
+# Geek Sessions Website
 
-This repos is used to hold the geek sessions website
+Repository for the [Geek Sessions website](https://geeksessionsio.geeksessions.now.sh/), whose intent is promoting tech-related events for professionals and enthusiasts alike.
 
-## Building and running on localhost
+## Project Setup
 
-First install dependencies:
+Prerequisites to either [run the project locally](#run-the-project-locally), or to [make a production build](#make-a-production-build).
+
+### Install required software
+
+To set up the project, you must first have these apps installed:
+
+- [Node.js (version LTS 12.16.2+)](https://nodejs.org/en/)
+  - includes NPM (version 6 or above)
+- [Now CLI][https://zeit.co/download] (to run the project locally)
+
+### Install and update project dependencies
+
+> **NOTE** we recommend you follow this step every time you start working on a new feature, as as dependencies may change over time
+
+Use npm to install and update project dependencies:
 
 ```sh
 npm install
 ```
 
-To create a production build:
+### Setup for local development
+
+To run the project locally, you need to follow two additional steps:
 
+1. install version 16 of [now CLI](https://zeit.co/download) globally:
 ```sh
-npm run build
+npm install -g now@16
 ```
+2. [link project to Google Calendar API](#set-up-google-calendar-API) (see instructions below) 
 
-To run in development
+### Set up Google Calendar API
 
-Install Now CLI
+To link the project to a Google Calendar account, **add a Google API Key** to a newly created **environment file**:
 
-```sh
-npm install -g now
+1. **get API key and id** for a Google calendar service
+   - to create or access this content, check [Google's "Creating a Google API Key" tutorial](https://docs.simplecalendar.io/google-api-key/)
```

Comment:
I think we should link to a tutorial on how to create and install API Keys, but I'm not sure if this one's adequate